### ISSUE TITLE
Hot pixels in maps

### ISSUE THREAD

**Issue by low-sky** [GBTAmmoniaSurvey/keystone#12]
Some maps seems to show hot pixels in the integrated intensity.  Verifying that the baseline fitting isn't going off the rails seems important.

**Comment by jakeown** [comment]
Yeah I have been noticing those as well.  I think they are just noisy pixels with bad fits that weren't weeded out by the update_NH3_mom0 function (I am using the PropertyMaps scripts from GAS).  I had issues with the default rebase function in the keystone pipeline, which kept fitting bad baselines.  That seemed to be due to the ammonia windows, which (I think) were overlapping with the emission lines in some cases where there are really broadened lines.  Instead, the rebaselining strategy I have been following is to fit a single order polynomial to the first and last ~200-300 channels, which appears to work pretty well.

**Comment by rfriesen** [comment]
We had/have similar issues in the GAS data; I was able to reduce the amount of hot pixels by masking based on the parameter map uncertainties in the updated moment maps script, but there is probably a better way to do this.

**Comment by jakeown** [comment]
I was leaning towards that route as well.  Or maybe increasing the 'threshold' parameter to only include channels with a sizeable brightness temp.  On a related note, I have noticed some low SNR pixels are being fit and have terrible fits, but they aren't weeded out with constraints I placed on their fitted parameters and uncertainties.  An example is attached.  The green line is the best-fit model for that spectrum. The best fit parameters and their uncertainties are printed atop the plot.  I was using a SNR cutoff of 3 in the line-fitting and this spectrum somehow squeaked through.  And it passed my liberal constraints on the best-fit parameters and their uncertainties.  Should I increase my lowest allowed excitation temperature threshold to get rid of these cases? 
![bad_fit](https://user-images.githubusercontent.com/16109704/35698197-38e481ca-0741-11e8-80f0-29f9041f32c5.png)

**Comment by keflavich** [comment]
You might also filter these out based on `max(model) < stddev_of_data`

**Comment by jakeown** [comment]
Ahh, ok. Like a SNR for the model. That sounds effective.
